tests/data/instances/sequence_tagging/verb_semantics_instance_test.py

```diff
@@ -0,0 +1,121 @@
+# pylint: disable=no-self-use,invalid-name
+
+from deep_qa.data import DataIndexer
+from deep_qa.data.instances.sequence_tagging.verb_semantics_instance import VerbSemanticsInstance
+
+import numpy
+from numpy.testing import assert_array_almost_equal
+from ....common.test_case import DeepQaTestCase
+
+class TestVerbSemanticsInstance(DeepQaTestCase):
+
+    def test_read_from_arguments(self):
+        sentence = ["roots", "absorb", "water"]
+        verb = (1, 1)
+        entity = (2, 2)
+        state_change = "MOVE"
+        arg1 = (-1, -1)
+        arg2 = (0, 0)
+
+        instance = VerbSemanticsInstance(sentence, verb, entity, state_change, arg1, arg2)
+
+        tags = ["B-ARG2", "O", "O"]
+
+        assert instance.sentence == sentence
+        assert instance.verb == verb
+        assert instance.entity == entity
+        assert instance.label[0] == state_change
+        assert instance.label[1] == tags
+
+    def test_read_from_line(self):
+        line_record = "roots####absorb####water\t1,1\t2,2\tMOVE\t-1,-1\t0,0"
+
+        instance = VerbSemanticsInstance.read_from_line(line_record)
+
+        sentence = ["roots", "absorb", "water"]
+        verb = (1, 1)
+        entity = (2, 2)
+        state_change = "MOVE"
+        tags = ["B-ARG2", "O", "O"]
+
+        assert instance.sentence == sentence
+        assert instance.verb == verb
+        assert instance.entity == entity
+        assert instance.label[0] == state_change
+        assert instance.label[1] == tags
+
+    def test_convert_instance_to_indexed_instance(self):
+        line_record = "roots####absorb####water\t1,1\t2,2\tMOVE\t-1,-1\t0,0"
+        data_indexer = DataIndexer()
+
+        instance = VerbSemanticsInstance.read_from_line(line_record)
+        roots_index = data_indexer.add_word_to_index("roots")
+        absorb_index = data_indexer.add_word_to_index("absorb")
+        water_index = data_indexer.add_word_to_index("water")
+
+        data_indexer.add_word_to_index("MOVE", namespace="state_changes")
+        data_indexer.add_word_to_index("O", namespace="tags")
+        data_indexer.add_word_to_index("B-ARG2", namespace="tags")
+
+        indexed_instance = instance.to_indexed_instance(data_indexer)
+        expected_sentence_indices = [roots_index, absorb_index, water_index]
+        expected_verb_indices = [0, 1, 0]
+        expected_entity_indices = [0, 0, 1]
+
+        assert_array_almost_equal(indexed_instance.sentence, expected_sentence_indices)
+        assert_array_almost_equal(indexed_instance.verb, expected_verb_indices)
+        assert_array_almost_equal(indexed_instance.entity, expected_entity_indices)
+
+        b_array = numpy.asarray([0, 1], dtype='int32')
+        o_array1 = numpy.asarray([1, 0], dtype='int32')
+        o_array2 = numpy.asarray([1, 0], dtype='int32')
```

Comment:
You don't need two `o_arrays` here, you could just have one, and use it twice on line 73 below.